src/icons/failed.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 21 21" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
There is a warning svg that is almost identical, we spoke with Tsvetan you should use the new one everywhere now.